cmscontrib/loaders/italy_yaml.py

```diff
@@ -219,6 +219,7 @@ def get_contest(self):
 
         tasks = load(conf, None, ["tasks", "problemi"])
         participations = load(conf, None, ["users", "utenti"])
+        participations = [] if participations is None else participations;
```

Comment:
```suggestion
        participations = [] if participations is None else participations
```
Semicolon is not needed here.

Reply:
Ups. Done.